website/public/locales/en/chat.json

```diff
@@ -4,10 +4,16 @@
   "config_title": "Chat configuration",
   "empty": "(empty)",
   "login_message": "To use this feature, you need to login again. Login using one of these providers:",
+  "max_new_tokens": "Max new tokens",
   "model": "Model",
   "preset": "Preset",
   "preset_custom": "Custom",
   "queue_info": "Your message is queued, you are at position {{ queuePosition, number, integer }} in the queue.",
+  "repetition_penalty": "Repetition penalty",
+  "temperature": "Temperature",
+  "top_k": "Top K",
+  "top_p": "Top P",
+  "typical_p": "Typical P",
```

Comment:
I'm not sure we want to translate this. For example, translating this to Vietnamese have completely no meaning at all

Reply:
These are very technical parameters, and even if we could translate them, that does not make them understandable, maybe this could give a short summary:

https://huggingface.co/docs/transformers/main_classes/text_generation#transformers.GenerationConfig.typical_p

I think that's ok like this for the first prototype.